shadowsocks-csharp/View/MenuViewController.cs

```diff
@@ -161,6 +170,20 @@ private void UpdateTrayIcon()
             Configuration config = controller.GetConfigurationCopy();
             bool enabled = config.enabled;
             bool global = config.global;
+
+            // set Windows 10 Theme color (1903+)
+            windowsThemeMode = getWindows10SystemThemeSetting();
+
+            switch (windowsThemeMode)

```

Comment:
Do we really need a switch case?

Reply:
Maybe Microsoft will add other theme features in Windows? And switch case can make future code more maintainable.

Reply:
Already achieved. :)